lib/ProductOpener/Ingredients.pm

```diff
@@ -4548,7 +4571,20 @@ sub normalize_fr_a_de_b ($a, $b) {
 	}
 }
 
-=head2 normalize_a_of_b ( $lc, $a, $b, $of_bool, $alternate_names_ref )
+# This function removes labels like "organic" from ingredients, so that we can check if they exist
+# with canonicalize_taxonomy_tag. The labels can be parsed out when doing ingredients analysis.
+
+sub remove_parsable_labels ($ingredients_lc, $ingredient) {
+	if ($ingredients_lc eq "en") {
+		$ingredient =~ s/(?:organic |fair trade )*//ig;
+	}
+	elsif ($ingredients_lc eq "fr") {
+		$ingredient =~ s/(?: bio| biologique| équitable|s|\s|' . $symbols_regexp . ')//ig;
+	}
```

Comment:
(future improvement) Maybe we could use a "used_in_ingredients" property in labels taxonomy to get them.

It's a bit of a pity we don't find this in german etc.

At least we could use taxonomy entries for organic and fair trade ?